### ISSUE TITLE
Why can't I use DNN and grammar together?

### ISSUE THREAD

**Issue by alexkinren** [julius-speech/julius#192]
When I try to use DNN-HMM and my implemented grammar together, I encounter an error while executing the following command.

julius -C /usr/local/src/julius/julius-kit/dictation-kit/am-dnn.jconf -dnnconf /usr/local/src/julius/julius-kit/dictation-kit/julius.dnnconf  -walign -nostrip -gram /usr/local/src/julius/dict_pa/pa -input rawfile

Error: voca_load_htkdict: line 1: triphone "*-p+a" or biphone "p+a" not found
Error: voca_load_htkdict: line 1: triphone "p-a+*" or biphone "p-a" not found
Error: voca_load_htkdict: the line content was: 0       [ぱ]    p a
Error: voca_load_htkdict: line 2: triphone "*-f+a" or biphone "f+a" not found
Error: voca_load_htkdict: line 2: triphone "f-a+*" or biphone "f-a" not found
Error: voca_load_htkdict: the line content was: 1       [ふぁ]  f a
Error: voca_load_htkdict: line 3: triphone "*-silB+*" or monophone "silB" not found
Error: voca_load_htkdict: the line content was: 2       [[s]]   silB
Error: voca_load_htkdict: line 4: triphone "*-silE+*" or monophone "silE" not found
Error: voca_load_htkdict: the line content was: 3       [[/s]]  silE
Error: voca_load_htkdict: begin missing phones
Error: voca_load_htkdict: *-f+a or biphone f+a
Error: voca_load_htkdict: *-p+a or biphone p+a
Error: voca_load_htkdict: *-silB+* or monophone silB
Error: voca_load_htkdict: *-silE+* or monophone silE
Error: voca_load_htkdict: f-a+* or biphone f-a
Error: voca_load_htkdict: p-a+* or biphone p-a
Error: voca_load_htkdict: end missing phones
Error: init_voca: error in reading /usr/local/src/julius/dict_pa/pa.dict: 4 words failed out of 0 words
ERROR: failed to read dictionary "/usr/local/src/julius/dict_pa/pa.dict"
ERROR: m_fusion: some error occured in reading grammars
ERROR: Error in loading model

**Comment by Estalhun** [comment]
Are all your text files UTF-8?

**Comment by alexkinren** [comment]
> Are all your text files UTF-8?
Yes, when my file contains Japanese characters, the encoding of the file is UTF-8.
pa.dfa:         text/plain; charset=us-ascii
pa.dfa.forward: text/plain; charset=us-ascii
pa.dict:        text/plain; charset=utf-8
pa.grammar:     text/plain; charset=us-ascii
pa.phone:       text/plain; charset=utf-8
pa.term:        text/plain; charset=us-ascii
pa.voca:        text/plain; charset=utf-8
pa.yomi:        text/plain; charset=utf-8
